xml_converter/src/packaging_protobin.cpp

```diff
@@ -284,3 +291,47 @@ void write_protobuf_file_per_map_id(
             &state);
     }
 }
+void _category_filter_by_depth(
```

Comment:
I dont think this function needs an underscore prefix just for being a non-public function.